src/newViews/UserGroup/AddUsergroupModal/index.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { useCallback } from 'react';
+import { isDefined } from '@togglecorp/fujs';
+import {
+    Modal,
+    Button,
+    PendingMessage,
+    TextInput,
+} from '@the-deep/deep-ui';
+import {
+    ObjectSchema,
+    PartialForm,
+    requiredCondition,
+    useForm,
+} from '@togglecorp/toggle-form';
+
+import { useLazyRequest } from '#utils/request';
+import NonFieldError from '#components/ui/NonFieldError';
+import _ts from '#ts';
+import styles from './styles.scss';
+
+interface UsergroupAdd {
+    title: string;
+    description?: string;
+}
+
+type FormType = {
+    id?: number;
+    title: string;
+};
+
+type FormSchema = ObjectSchema<PartialForm<FormType>>;
+type FormSchemaFields = ReturnType<FormSchema['fields']>;
+
+const schema: FormSchema = {
+    fields: (): FormSchemaFields => ({
+        title: [requiredCondition],
+    }),
+};
+
+const defaultFormValue: PartialForm<FormType> = {};
+
+interface Props {
+    onModalClose: () => void;
+    onTableReload: () => void;
```

Comment:
Shouldn't the prop name be something different.
The AddUsergroupModal shouldn't know about a sibling Table that needs reload